client/src/views/pages/banned.js

```diff
@@ -0,0 +1,556 @@
+                ].map((organisation, key) => (
+                  <li key={key}>
+                    <a href={organisation.discord} target="_blank" rel="noopener noreferrer">
+                      {organisation.name}
+                    </a>
+                  </li>
+                ))}
+              </ul>
+            </>
+          )}
+        </Col>
+      </Row>
+      <Row className="mt-4">
+        <Col className="text-left">
+          <Button color="default" onClick={props.previousStep}>
+            <i className="fa fa-arrow-left mr-2" />
+            Back
+          </Button>
+        </Col>
+        <Col className="text-right">
+          <Button color="primary" onClick={props.nextStep}>
+            Next
+            <i className="fa fa-arrow-right ml-2" />
+          </Button>
+        </Col>
+      </Row>
+    </>
+  );
+}
+
+function GettingUnbannedFromTheSquadCommunityBanList(props) {
+  return (
+    <>
+      <Row>
+        <Col>
+          <h3 className="text-center">Getting unbanned from the Squad Community Ban List</h3>
+          <p>
+            Squad servers may opt to use a Squad Community Ban List export ban list to protect their
+            community. If you have been kicked from a server with a reason relating to the Squad
+            Community Ban List then it means that you are probably met the criteria to be listed on
+            one of these export ban lists. Below we will explain this criteria and explain how you
+            can attempt to get removed from our export ban lists.
+          </p>
+          <h6>Criteria System</h6>
+          <p>
+            Our criteria system uses a point based system that each Squad server can configure when
+            creating an export ban list. Bans listed on our website will contribute a number of
+            points which can be configured to be different depending on whether they were are active
+            or expired, their age and who's ban list they belong to. Once the total number of points
+            a player has exceeds to configurable threshold they will be added to the export ban list
+            and banned from the server as a result.
+          </p>
+          <p>
+            Considering the following example using our default points. Bob has 2 active bans that
+            contribute 3 points each and 4 expired bans that contribute 1 point each. This makes his
+            total number of points 10 which exceeds the default threshold of 9 and therefore Bob
+            will be added to the export ban list.
+          </p>
+          <h6>Getting Unbanned from the Squad Community Ban List</h6>
+          <p>
+            There are a number of different ways of getting unbanned from the Squad Community Ban
+            List:
+          </p>
+          <ul className="font-weight-light">
+            <li>
+              Appeal the bans listed on our website so that you no longer meet the criteria of our
```

Comment:
I think repeating that they should appeal at the banning communities can't hurt.... Else they'll try to appeal on the Discord.
```suggestion
              Appeal the bans (to the banning communities) listed on our website so that you no longer meet the criteria of our
```